src/visit.ts

```diff
@@ -334,13 +345,9 @@ export function visitEachChild<T extends FunctionlessNode>(
     const _else = node._else ? visitor(node._else) : undefined;
 
     ensure(when, isExpr, "a IfStmt's when must be an Expr");
-    ensure(then, isBlockStmt, "a IfStmt's then must be a BlockStmt");
+    ensure(then, isStmt, "a IfStmt's then must be a BlockStmt");
```

Comment:
Why did the guard change but the error still says it must be a block? What's changed here?

Reply:
if only we generated those errors instead of hand wrote them, haha...